pkg/acsengine/params_k8s.go

```diff
@@ -148,7 +148,7 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
 			}
 			if kubernetesConfig.IsDashboardEnabled() {
 				dashboardAddon := kubernetesConfig.GetAddonByName(DefaultDashboardAddonName)
-				dashboardAddon.GetAddonContainersIndexByName(DefaultDashboardAddonName)
+				c := dashboardAddon.GetAddonContainersIndexByName(DefaultDashboardAddonName)
```

Comment:
gah

can we change the name of var `c` above? If that wasn't `c` then we wouldn't gotten a compile time error and this bug would never have happened

Reply:
Sure. I can do that for other add-ons as well. In this instance, ill rename it to dashboardIn

Reply:
👍